### PR TITLE
feat: new documentation information architecture

### PR DESCRIPTION
**Pull request by talagluck**
Starting to migrate from the current structure to the structure described here: https://github.com/GlareDB/meta/blob/main/notes/2024-01-09.docs-information-architecture.org

We will use this as a feature branch, and merge any docs that are newly written for the re-arch into this branch, then merge this branch in the next week or two.

### CODE REVIEW

**Comment by tychoish** [comment]
do we have previews here?

**Comment by talagluck** [comment]
I've been working from a local live-render. @greyscaled any idea what it would take to have previews like we do for glaredb.com?

File: introduction/installation/sign-up-for-cloud.md [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
-title: Sign Up for Cloud
+title: Sign Up for GlareDB Cloud
 layout: default
 nav_order: 1
 parent: Installation
 grand_parent: Introduction / Getting Started
 ---
 
-## Sign Up for Cloud
+# Sign Up for GlareDB Cloud
```
Comment (tychoish) [inline introduction/installation/sign-up-for-cloud.md:9]: I'm not sure I understand the reason for/impact of the heading change

File: introduction/installation/sign-up-for-cloud.md [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
+
+[GlareDB Cloud] is a fully-managed offering of GlareDB that scales with your
+workloads and teams. It is free to get started, and signing up is quick!
+
+![Sign-in]
+
+<!-- TODO: link to pages once they're written -->
+
+After signing up, an Organization with a fully-managed GlareDB Deployment will be
+created and ready to use with the **SQL Workspace** editor in your browser.
```
Comment (tychoish) [inline introduction/installation/sign-up-for-cloud.md:19]: make the sentence active:

Signing up for cloud will create a deployment [what it is] within an organization [what it is].

I might also put the serverless word here.

File: introduction/installation/sign-up-for-cloud.md (hunk 1 above)
Comment (greyscaled) [reply introduction/installation/sign-up-for-cloud.md:9]: The meta at the top of the file isn't a heading, so making the top-most heading of the page an `h2` was just a mistake

File: introduction/installation/sign-up-for-cloud.md (hunk 2 above)
Comment (greyscaled) [reply introduction/installation/sign-up-for-cloud.md:19]: There are comments in the markdown - there are links missing for the Deployment and Organization that will be added.

File: introduction/installation/python-bindings.md [hunk 3]
```diff
@@ -6,4 +6,242 @@ parent: Installation
 grand_parent: Getting Started
 ---
 
-## Serverless
+# Python Bindings
+
+GlareDB can be used in Python environments along with [Pandas] and [Polars] to
+work with distributed data.
+
+For more information, refer to our in-depth [blog post Working with Python].
```
Comment (tychoish) [inline introduction/installation/python-bindings.md]: reference should probably have quotes

File: introduction/key-concepts-in-glaredb.md [hunk 4]
```diff
@@ -1,20 +1,149 @@
...
-### Table Functions
+## Table Functions
 
-### External Data Sources
+A core tenet of GlareDB is that [All Data are SQL Addressable], even data not
+in a SQL database.
 
-### Native Data Storage
+Table functions access data, on-demand, wherever it is, including data sources
+that have not been added to GlareDB.
```
Comment (tychoish) [inline introduction/key-concepts-in-glaredb.md:16]: "not added to glaredb" is sort of confusing? I think you mean "registered as an external table" but it makes it sound like an implementation deficiency

**Review by talagluck — COMMENTED** [review]
@tychoish did the storage doc get merged prematurely?

**Pull request merged**